bin/use-client-pgbouncer

```diff
@@ -16,23 +16,31 @@
 
 set +e
 
-export PGBOUNCER_QUIET=${PGBOUNCER_QUIET:-false}
+function inf() {
+  echo -e "INFO:  $*"
+}
+
+function err() {
+   echo -e "ERROR: $*"
+}
+
+export PGBOUNCER_OUTPUT_URLS=${PGBOUNCER_OUTPUT_URLS:-false}
 
 function is-enabled() {
   ( shopt -s extglob nocasematch
     [[ $1 == @(1|true|yes|on) ]]
   )
 }
 
-function check-pgbouncer-is-enabled() {
+function is-pgbouncer-service-enabled() {
   if ! is-enabled "${PGBOUNCER_ENABLED}"; then
-    echo pgbouncer-disabled
+    inf pgbouncer-disabled
     return 1
   fi
   return 0
 }
```

Comment:
This look like the same implementation above, I would abstract it into a simple reusable function.